pkg/mattermost/database_mysql.go

```diff
@@ -0,0 +1,94 @@
+package mattermost
+
+import (
+	"fmt"
+
+	mattermostv1beta1 "github.com/mattermost/mattermost-operator/apis/mattermost/v1beta1"
+	"github.com/mattermost/mattermost-operator/pkg/components/utils"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type MySQLDBConfig struct {
+	secretName   string
+	rootPassword string
+	userName     string
+	userPassword string
+	databaseName string
+}
+
+func NewMySQLDB(secret corev1.Secret) (*MySQLDBConfig, error) {
```

Comment:
```suggestion
func New MySQLDBConfig(secret corev1.Secret) (*MySQLDBConfig, error) {
```

Same comment for this one